contracts/core/extensions/CoreInternal.sol

```diff
@@ -142,7 +142,7 @@ contract CoreInternal is
      * @param  _priceLibrary   Address of contract Price Library to enable or disable
      * @param  _enabled        Whether the pricing library is enabled for use in proposal cycle
      */
-    function setPriceLibraryEnabled(
+    function registerPriceLibrary(
         address _priceLibrary,
         bool _enabled
```

Comment:
Same comment. Why are we adding the `bool` enabled? Why do we need it?

Reply:
https://github.com/SetProtocol/set-protocol-contracts/pull/254

Reply:
Cool - as long as we have an event emitted, this works